downstream/assemblies/platform/assembly-installing-controller-operator-external-db.adoc

```diff
@@ -8,48 +8,49 @@ ifdef::context[:parent-context: {context}]
 
 
 [id="installing-contr-operator-external"]
-= Installing {ControllerName} on {OCP} with an external database and routed web traffic
+= Installing {ControllerName} on {OCP} with an external database
 
 
 :context: installing-contr-operator-external
 
 
 [role="_abstract"]
-You can use these instructions to install the {HubName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
+You can use these instructions to install the {ControllerName} operator on {OCP} with an external PostgreSQL 12 database and route, as well as specify customer resources.
 
 // mirrors AWX operator flow
 
 == Prerequisites
 
 * You have installed the catalog containing the {PlatformName} operators
```

Comment:
Should this be "You have installed the {PlatformName} operator" ?

Also, it might be an idea to break the procedural content in the assemblies out into separate procedure and concept modules.